projects/sdr_math/README.md

```diff
@@ -0,0 +1,10 @@
+# Purpose of the Code
+This code is intended to run simulations to evaluate the properties of sparsely distributed representations (SDRs) with an emphasis on using SDRs for machine learning tasks. The figures generated are automatically saved as PDF files. The properties of SDRs were discussed in the paper "How Can We Be So Dense? The Benefits of Using Highly Sparse Representations" availalbe at: https://arxiv.org/abs/1903.11257.
+
```

Comment:
Could you also include a reference to this paper:

https://arxiv.org/abs/1601.00720

Reply:
Yes, I added a reference to the paper.